src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -75,4 +78,38 @@ trait CromwellApiService extends HttpService with PerRequestCreator  {
           complete(StatusCodes.BadRequest)
       }
     }
+
+
+  @ApiOperation(value = "Submit a new workflow for execution",
+    nickname = "submit",
+    httpMethod = "POST",
+    produces = "application/json"
+//  ,response = classOf[WorkflowSubmitResponse]
+  )
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "wdlSource", required = true, dataType = "string", paramType = "form"),
+    new ApiImplicitParam(name = "workflowInputs", required = true, dataType = "string", paramType = "form")
+  ))
+  @ApiResponses(Array(
+    new ApiResponse(code = 201, message = "Successful Request"),
+    new ApiResponse(code = 400, message = "Malformed Input"),
+    new ApiResponse(code = 500, message = "Internal Error")
+  ))
+  def submitRoute =
+    path("workflows") {
+      post {
+        formFields("wdlSource", "workflowInputs") { (wdlSource, workflowInputs) =>
+
+          val tryMap = Try(workflowInputs.parseJson.convertTo[Map[String, String]])
```

Comment:
I'll take care of this with the merge into my PR, which will handle this a little better.